input/pagecontent/index.md

```diff
@@ -106,7 +106,7 @@ The [VaccinationCredentialPatient] and [Covid19LaboratoryBundle]/[InfectiousDise
 
 ```json
 "meta": {
-  "security": [{"code": "IAL1"}]
+  "security": [{"code": "IAL1", "system": "http://terminology.hl7.org/CodeSystem/loa"}]
```

Comment:
Need to use the new URI for system

Reply:
Also change IAL1 to one of the other values in that value set

Reply:
updated, thanks!